kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/ConversationActivity.java

```diff
@@ -1239,9 +1239,8 @@ public void processAttachment() {
         }
     }
 
-    public void processImageVideo() {
-        Intent contentChooserIntent;
-        contentChooserIntent = FileUtils.createGetContentIntent(FileUtils.GalleryFilterOptions.IMAGE_VIDEO, getPackageManager());
+    public void processMultiSelectGallery() {
+        Intent contentChooserIntent = FileUtils.createGetContentIntent(FileUtils.GalleryFilterOptions.IMAGE_VIDEO, getPackageManager());
```

Comment:
@shubhamtewari can we rename IMAGE_VIDEO as well?

Reply:
yes, didn't see that